recipes/vscode-python/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "vscode-python" %}
+{% set version = "2021.2.582707922" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: https://github.com/microsoft/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 517360d2af77dc7376d02cdfdf9348142d1d2c7b792e5ff6c548871b2aa4e991
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - nodejs 12.14.1
+    - code-server >=3.9
+  host:
+    - nodejs 12.14.1
+    - code-server >=3.9
+  run:
+    - code-server >=3.9
+    - python >=2.7
+    - jedi <0.18
+    - pygls  # [py>=36]
+    - jedi-language-server <0.22  # [py>=36]
+    - isort 5.7.0  # [py>=36]
```

As with the other vscode plugin PR: Why are they listed here? How did it work when installing the `vsix` directly instead of using a conda package?

```suggestion
    - python >=2.7
    - jedi <0.18
    - pygls  # [py>=36]
    - jedi-language-server <0.22  # [py>=36]
    - isort 5.7.0  # [py>=36]
```

---

Well, they have this [`requirements.in`](https://github.com/microsoft/vscode-python/blob/main/requirements.in) floating around in the repo without really mentioning what it is for. They also don't mention it in the [contributing guide](https://github.com/microsoft/vscode-python/wiki/Contributing). I guess it's worth a shot to remove them and see what happens.